Gruntfile.js

```diff
@@ -45,16 +45,28 @@ module.exports = function(grunt) {
 
   var env = grunt.option("env") || "release";
 
+  var pkg = grunt.file.readJSON("package.json");
```

Comment:
Should use `var pkg = require('package.json')` at the top of the file instead.

Reply:
I would also check at the top, right after this line if a local options directory exists;
`var hasOptionsDirectory = fs.existsSync('options')`

Reply:
Awesome. Didn't know you could use `require` in that situation.